pom.xml

```
@@ -3,12 +3,12 @@
   <groupId>oryanmoshe.kafka.connect.util</groupId>
   <artifactId>TimestampConverter</artifactId>
   <packaging>jar</packaging>
-  <version>1.2.2-SNAPSHOT</version>
+  <version>1.2.3-SNAPSHOT</version>
   <name>TimestampConverter</name>
   <url>http://maven.apache.org</url>
 
   <properties>
-    <version.debezium>1.1.0.Final</version.debezium>
+    <version.debezium>1.2.2.Final</version.debezium>
```

```suggestion
    <version.debezium>1.3.1.Final</version.debezium>
```
Why not use the latest version?